pkg/kubelet/util/manager/cache_based_manager.go

```diff
@@ -92,7 +92,7 @@ func isObjectOlder(newObject, oldObject runtime.Object) bool {
 	return newVersion < oldVersion
 }
 
-func (s *objectStore) AddReference(namespace, name string) {
+func (s *objectStore) AddReference(namespace, name string, podUID types.UID) {
```

Comment:
Hmm - this doesn't seem to be used anywhere...

Reply:
I misunderstand your meaning...

Will record this in objectStore next commit.

Reply:
> I misunderstand your meaning...

Ahh sorry. I meant everything related to this change, (to separate it from the change that was introducing pod-specific stuff).